barcode/writer.py

```diff
@@ -112,8 +113,10 @@ def calculate_size(self, modules_per_line, number_of_lines, dpi=300):
         """
         width = 2 * self.quiet_zone + modules_per_line * self.module_width
         height = 2.0 + self.module_height * number_of_lines
+        number_of_text_lines = self.text.count('\n') + 1
```

Comment:
This is probably clearer
```suggestion
        number_of_text_lines = len(self.text.splitlines())
```